controllers/expense.controller.js

```diff
@@ -0,0 +1,21 @@
+const { expanse } = require("../models");
+
+
+exports.add = async (req, res) => {
+  const { shop } = req.body;
+  const { cost } = req.body;
+  if (!shop || !cost) {
+    res.status(422);
+    return res.send({answer: "Shop name or price input is not defined."});
+  }  
+  if (cost < 0 && isNaN(cost)) {
```

Comment:
please separately validate shop and cost and return meaningful separate errors for each field